packages/hoppscotch-common/src/components/accessTokens/index.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <AccessTokensOverview
+    @show-access-tokens-generate-modal="showAccessTokensGenerateModal = true"
+  />
+
+  <AccessTokensList
+    :access-tokens="accessTokens"
+    :has-more-tokens="hasMoreTokens"
+    :tokens-list-loading="tokensListLoading"
+    @delete-access-token="displayDeleteAccessTokenConfirmationModal"
+    @fetch-more-tokens="fetchAccessTokens"
+  />
+
```

Comment:
should we add an empty state UI ?

Reply:
Had an initial thought that the overview section comprising the `Generate new token` action would serve the purpose. But, it's better to stay consistent with the pattern followed across list views where the error and empty states are displayed; updated in https://github.com/hoppscotch/hoppscotch/pull/4094/commits/782b722303f034a461653e6ca9e3ce4d79492292.